examples/benchmarks/bubblebench/simple-tree/package.json

```diff
@@ -66,7 +66,7 @@
 		"@fluidframework/eslint-config-fluid": "^5.1.0",
 		"@fluidframework/test-tools": "^1.0.195075",
 		"@types/jest": "29.5.3",
-		"@types/jest-environment-puppeteer": "workspace:",
+		"@types/jest-environment-puppeteer": "workspace:~",
```

Comment:
this is odd. Any idea what's up with it?
Seems odd that we would have a workspace version of "@types/jest-environment-puppeteer" in the first place, and also odd that this PR would change how we depend on it.

Reply:
There was a similar change in the 4.0 bump PR: https://github.com/microsoft/FluidFramework/pull/20661#discussion_r1567764270

I don't know the cause. But we definitely have an in-repo "copy" of the types package. See here:

https://github.com/microsoft/FluidFramework/blob/442cb76c288ca75919d71f3c0e1e3f5d4263ebbe/packages/test/types_jest-environment-puppeteer/index.d.ts#L6-L9

Reply:
Sorry, I missed that this was package.json, not the lockfile. The old dependency was manually added, probably, and we normalize all the workspace deps as part of bumping. So this change and the corresponding lockfile change is correct.